src/SAWScript/Prover/Exporter.hs

```diff
@@ -297,7 +316,10 @@ writeCoqTerm ::
   Term ->
   TopLevel ()
 writeCoqTerm name notations skips path t = do
-  let configuration = coqTranslationConfiguration notations skips
+  let configuration =
+        withImportSAWCorePrelude $
+        withImportCryptolPrimitivesForSAWCore $
```

Comment:
As far as I can tell, these imports weren't required before this patch. Is there a reason that they are needed now?

Reply:
The SAWCorePrelude used to be added via the extraPreamble.

The CryptolPrimitivesForSAWCore was not added, and is seemingly not needed when one run `write_coq_{...}` commands (at least I have not needed it), but is needed when running the `offline_coq` tactic from SAW script when it is working with Cryptol code.

I likely need to revisit exactly what imports are needed for what commands, and where the line is between imports that are mandatory and ones that are only needed when working with certain types of files. But if this PR is blocking others, I would be happy getting back to this decision in a subsequent one.

Reply:
> The SAWCorePrelude used to be added via the extraPreamble.

True, but the `extraPremable` was only used in another function (`writeCoqCryptolPrimitivesForSAWCore`), not this function (`writeCoqTerm`), right?

> The CryptolPrimitivesForSAWCore was not added, and is seemingly not needed when one run `write_coq_{...}` commands (at least I have not needed it), but is needed when running the `offline_coq` tactic from SAW script when it is working with Cryptol code.

Ah, I think I assumed that this patch intended to simply update the `saw-core` submodule without affecting any functionality—I only read the commit message, not the PR description. But the PR description does indeed mention the change to `offline_coq`. In that case, I'll defer to your judgment.